server/src/main/scala/cromwell/CromwellEntryPoint.scala

```diff
@@ -183,6 +193,7 @@ object CromwellEntryPoint extends GracefulStopSupport {
   def validateRunArguments(args: CommandLineArguments): WorkflowSourceFilesCollection = {
     val sourceFileCollection = (args.validateSubmission(EntryPointLogger), writeableMetadataPath(args.metadataOutput)) mapN {
       case (ValidSubmission(w, r, i, o, l, Some(z)), _) =>
+        //noinspection RedundantDefaultArgument
```

Comment:
What does this mean?

Reply:
The `workflowOnHold = false` arguments are producing IntelliJ warnings. To fix the warnings one can either tell IntelliJ to ignore the redundant default argument, or remove passing in the default. I chose the former.

https://github.com/JetBrains/intellij-scala/blob/fc53da69ea839506c733e43a802f92dcbb86a62e/scala/scala-impl/resources/META-INF/scala-plugin-common.xml#L1032-L1035